### PR TITLE
Improve search behaviour

### PR DESCRIPTION
**Pull request by serg**

### CODE REVIEW

File: themes/hugo-theme-learn/static/js/search.js [hunk 1]
```diff
@@ -1,89 +1,128 @@
...
+  var getSearchOptions = function(searchResultsBox) {
+    var searchFirstRun = true;
 
-function endsWith(str, suffix) {
-  return str.indexOf(suffix, str.length - suffix.length) !== -1;
-}
+    var options = {
+      searchClient: algoliasearch(
+        'Q03DAQELCN',
+        'f8b7a0497068023efe046671dd40ec62'
```
Comment (xmudrii) [inline themes/hugo-theme-learn/static/js/search.js:8]: What is this? If this is an API key, should we really put it publicly in the source code?

Comment (serg) [reply themes/hugo-theme-learn/static/js/search.js:8]: This is public key for using search API

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**